frame/vesting/src/tests.rs

```diff
@@ -96,50 +96,41 @@ fn check_vesting_status() {
 	});
 }
 
-#[test]
+#[test]	
 fn check_vesting_status_for_multi_schedule_account() {
-	ExtBuilder::default().existential_deposit(ED).build().execute_with(|| {
+	ExtBuilder::default().existential_deposit(ED).add_vest(2, ED * 10, ED, 0)
+	.add_vest(2, ED * 30, ED, 5).build().execute_with(|| {
 		assert_eq!(System::block_number(), 1);
 		let sched0 = VestingInfo::new(
 			ED * 20,
 			ED, // Vesting over 20 blocks
 			10,
 		);
-		// Account 2 already has a vesting schedule.
-		assert_eq!(Vesting::vesting(&2).unwrap(), vec![sched0]);
-
-		// Account 2's free balance is from sched0.
-		let free_balance = Balances::free_balance(&2);
-		assert_eq!(free_balance, ED * (20));
-		assert_eq!(Vesting::vesting_balance(&2), Some(free_balance));
 
-		// Add a 2nd schedule that is already unlocking by block #1.
 		let sched1 = VestingInfo::new(
 			ED * 10,
 			ED, // Vesting over 10 blocks
 			0,
 		);
-		assert_ok!(Vesting::vested_transfer(Some(4).into(), 2, sched1));
-		// Free balance is equal to the two existing schedules total amount.
-		let free_balance = Balances::free_balance(&2);
-		assert_eq!(free_balance, ED * (10 + 20));
-		// The most recently added schedule exists.
-		assert_eq!(Vesting::vesting(&2).unwrap(), vec![sched0, sched1]);
-		// sched1 has free funds at block #1, but nothing else.
-		assert_eq!(Vesting::vesting_balance(&2), Some(free_balance - sched1.per_block()));
 
-		// Add a 3rd schedule.
 		let sched2 = VestingInfo::new(
 			ED * 30,
 			ED, // Vesting over 30 blocks
 			5,
 		);
-		assert_ok!(Vesting::vested_transfer(Some(4).into(), 2, sched2));
 
-		System::set_block_number(9);
+		// Account 2 already has 3 vesting schedules.
+		assert_eq!(Vesting::vesting(&2).unwrap(), vec![sched0, sched1, sched2]);
```

Comment:
Hi @ggwpez, I had added extra assertions here. Or do you mean I should do something like:

```
// Account 2 has more than 1 schedule.
assert_ne!(Vesting::vesting(&2).unwrap(), vec![sched0]);
```